src/app/participant-list/participant-list.component.ts

```diff
@@ -1688,7 +1688,7 @@ export class ParticipantListComponent implements OnInit {
           }
           if (this.assignTissue) {
             if (this.assignee.assigneeId === "-1") {
-              pt.participant.assigneeMr = null;
+              pt.participant.assigneeTissue = null;
```

Comment:
I think this needed correction because we are removing the tissue assignee here based on the if L1690